packages/bento-design-system/src/util/LocalizedString.ts

```diff
@@ -1,9 +1,8 @@
-/**
- * A string that has been localized
- */
-export type LocalizedString = string & {
-  readonly LocalizedString: "LocalizedString";
-};
+import { ConfiguredTypes } from "./ConfigurableTypes";
+
+export type LocalizedString = ConfiguredTypes["LocalizedString"];
```

Comment:
Note to self, maybe it's better to do something like `(string | number) & ConfiguredTypes["LocalizedString"]` so that we have a baseline to work with. Otherwise a client can potentially override this with a unrelated type and then the whole thing becomes unsafe